### PR TITLE
Add models with tests

### PR DESCRIPTION
**Pull request by erikpivotal**

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Item < ApplicationRecord
+  belongs_to :retro
+  validates :retro_id, presence: true
```
Comment (ghost) [inline app/models/item.rb]: I would group the validates together and the relationships together...small file now but easier to find when they are grouped together

also white space...

File: spec/models/item_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    end
+    it 'is invalid if no retro_id' do
+      item = Item.new(emotion_type_id: emotion_type.id, content: 'blah')
+      expect(item).to_not be_valid
+      expect(item.errors['retro_id']).to include "can't be blank"
+    end
+    it 'is invalid if no content' do
+      item = Item.new(emotion_type_id: emotion_type.id, retro_id: retro.id)
+      expect(item).to_not be_valid
+      expect(item.errors['content']).to include "can't be blank"
```
Comment (ghost) [inline spec/models/item_spec.rb]: missing the uniqueness validation for content

**Review by ghost — CHANGES_REQUESTED** [review]

File: spec/models/item_spec.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    it 'is invalid if no content' do
+      item = build(:item, content: nil)
+      expect(item).to_not be_valid
+      expect(item.errors['content']).to include "can't be blank"
+    end
+
+    it 'is invalid if duplicate content' do
+      create(:item, content: 'ABC')
+      item = build(:item, content: 'ABC')
```
Comment (ghost) [inline spec/models/item_spec.rb]: i would use this pattern
existing = create(:item)
item = build(:item, content: existing.content)

File: spec/models/item_spec.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      item.errors.inspect
```
Comment (ghost) [inline spec/models/item_spec.rb]: hmm what is this line for?

**Review by ghost — CHANGES_REQUESTED** [review]

File: spec/models/item_spec.rb (hunk 4 above)
Comment (erikpivotal) [reply spec/models/item_spec.rb]: Accidentally left it in there while checking errors

**Pull request merged**